packages/common/src/tools/queue/Retries.ts

```diff
@@ -0,0 +1,29 @@
+import { ShouldRetry } from './types'
+
+function always() {
+  return { retry: true }
+}
+
+function exponentialBackOff<T>(stepMs: number): ShouldRetry<T> {
```

Comment:
Maybe an optional maximum would be nice here.

Reply:
Do you mean maximum waiting time?